slices/openjdk-21-jre-headless.yaml

```diff
@@ -0,0 +1,215 @@
+package: openjdk-21-jre-headless
+
+essential:
+  - openjdk-21-jre-headless_copyright
+
+slices:
+
+  all:
```

Comment:
`all` is too ambiguous, as it could mean the whole package or `all` the slices in this SDF...meaning that it is hard to predict what you'll get out of it and that it may easily change over time. If you don't need this slice for your use case, I'd recommend not including it. Otherwise, let's maybe call it something a bit safer...like  `standard` perhaps? We use that for python - it's less comprehensive than `all` and still conveys a notion of something fuller than `core`.